packages/app-store/_utils/nodeScheduler.ts

```diff
@@ -0,0 +1,298 @@
+import type { Prisma } from "@prisma/client";
+import schedule from "node-schedule";
```

Comment:
Just a sidenote. We're planning to migrate away from "node-schedule" aren't we? Not a blocker but something to have in mind as a follow up. cc @alannnc @CarinaWolli

Reply:
There is already a PR up for replacing node-schedule: https://github.com/calcom/cal.com/pull/10946

I will review that today and can help resolving conflicts